app/controllers/sessions/magic_links_controller.rb

```diff
@@ -22,11 +21,41 @@ def code
       params.expect(:code)
     end
 
+    def respond_to_valid_code_from(magic_link)
+      respond_to do |format|
+        format.html do
+          start_new_session_for magic_link.identity
+          redirect_to after_sign_in_url(magic_link)
+        end
+
+        format.json do
+          new_access_token = magic_link.identity.access_tokens.create!(permission: :write)
+          render json: { access_token: new_access_token.token }
```

Comment:
Jay added an identity endpoint that returns the email, account, user and other info but using a jbuilder template so we get to levarage caching and control the response payload.

So I removed those fields from here. I know it's now two requests to get everything, hope that doesn't cause too many problems.